engine/api/v2_project_vcs.go

```diff
@@ -234,8 +236,62 @@ func (api *API) getVCSProjectHandler() ([]service.RbacChecker, service.Handler)
 			}
 			vcsProject.Auth.Username = vcsClear.Auth.Username
 			vcsProject.Auth.SSHKeyName = vcsClear.Auth.SSHKeyName
+			vcsProject.Auth.GPGKeyName = vcsClear.Auth.GPGKeyName
+			vcsProject.Auth.EmailAddress = vcsClear.Auth.EmailAddress
 			vcsProject.Auth.SSHUsername = vcsClear.Auth.SSHUsername
 			vcsProject.Auth.SSHPort = vcsClear.Auth.SSHPort
 			return service.WriteMarshal(w, r, vcsProject, http.StatusOK)
 		}
 }
+
+func (api *API) GetVCSPGKeyHandler() ([]service.RbacChecker, service.Handler) {
+	return nil,
```

Comment:
RBAC ?

Reply:
no RBAC rule for /v2/vcs/gpgkeys/{gpgKeyID}
like /v2/user/gpgkey/{gpgKeyID} or /v2/user/{user}/gpgkey

Reply:
Ack